source/_layouts/default.html

```diff
@@ -55,7 +55,7 @@
 
     <div id="doc-switchers" class="container">
       <div class="col-md-8">{% include template/doc_switcher.html %} {% include template/version_switcher.html %}</div>
-      <div class="col-md-4"></div>
+      <div class="col-md-4"><div class="visible-sm-inline visible-xs-inline"><a href="#document-nav">Jump to table of contents</a></div></div>
```

Comment:
Since this jumps to the document nav instead of the page nav, "table of contents" seems ambiguous. Could this use a new variable to be more specific, like 

`Jump to {{product.title}} menu`

or

`Jump to {% if page.url contains "/puppetserver/" %}Puppet Server{% elif page.url contains "/puppetdb/" %}PuppetDB{% endif %} menu` ... etc.

Also, to mirror the "Back to top" link at the bottom and indicate what this does, could this include a downward arrow symbol (↓ `&#8595;`) instead of "Jump to"? That could make it a little more localization friendly, too.